Wait for engine 6

Aavild opened the pull request:
Included more clear running steps with a simple start config and which JDK to use

Made it easier to define parameters as a sequence of parameters instead of a single parameter.
Reason: Beforehand if one wanted to use 2 parameters and tried making parameterExpressions: "serve 127.0.0.1:7000" it was run as exactly: `./reveaal "serve 127.0.0.1:7000"` rather than `./reveaal "serve" "127.0.0.1:7000"`

Stalls tests till the engine has printed something. 
Reason: Beforehand usually around 3.500 out of the 4.000 tests in the defaultconfig ran before the engines could start up.

frederikja163 reviewed (commented):
Internal approval

williamwoldum reviewed (changes requested):
Lige nu peger jeres submodule pointer på et gammelt commit `ae8364e6c8942a5cfde24adec779ca87d4431e4a`

I bør:
- `cd src/main/Ecdar-ProtoBuf`
- `git checkout 1e9b849818df8eec0f15ec2ce9a28b751411bf4b`
- `git commit -m "updated submodule pointer`
- `git push`

Måske også merge med `main` inden push

<3

Aavild commented:
> Lige nu peger jeres submodule på et gammelt commit `ae8364e6c8942a5cfde24adec779ca87d4431e4a`.
> 
>     * `git merge main`
> 
>     * `cd src/main/Ecdar-ProtoBuf`
> 
>     * `git fetch`
> 
>     * `git checkout 1e9b849818df8eec0f15ec2ce9a28b751411bf4b`
> 
>     * `cd ..`
> 
>     * `git commit -m "updated submodule pointer"`
> 
>     * `git push`
> 
> 
> <3

Fixed the pointer by merging main into the branch. I would argue that that this would be solved automatically because the PR doesn't try to merge the outdated submodule back into the main branch(no merge conflicts and changed files)
![image](https://github.com/ECDAR-AAU-SW-P5/Ecdar-Test/assets/46568674/cd5ab83a-e467-4013-9bef-0fc066ea7bac)
